package.json

```
@@ -38,5 +38,8 @@
     "eslint-plugin-promise": "4.2.1",
     "eslint-plugin-react": "7.14.3",
     "json-diff": "0.5.4"
+  },
+  "publishConfig": {
+    "registry": "https://nexus.ci.diabeloop.eu/repository/npm-diabeloop/"
```

here we have to use the npm host on nexus
```suggestion
    "registry": "https://nexus.ci.diabeloop.eu/repository/your-loops-npm/"
```